tests/test_postgres_options_plugin.py

```
@@ -1,6 +1,7 @@
 """Test behavior of postgres_options passed in different ways."""
 
 from pathlib import Path
+from typing import Iterator
```

Ruff (0.16.0)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)